src/main/java/org/generationcp/breeding/manager/customcomponent/RemoveSelectedGermplasmAsDialog.java

```diff
@@ -209,6 +217,34 @@ public void setCancelButton(final Button cancelButton) {
 
 		@Override
 		public void buttonClick(final Button.ClickEvent event) {
+			final Set<Integer> germplasmUsedInOtherLists = this.removeSelectedGermplasmAsDialog.getGermplasmPresentInOtherLists(
```

Comment:
Remove the second warning here also. 
Make sure we are not adding new code, especially in MW and BMSAPI, just to cater to this Vaadin screen which we will soon phase out.